### ISSUE TITLE
Wrong placement in Noto sans arabic & UI

### ISSUE THREAD

**Issue by pupyc** [googlei18n/noto-source#113]
Character: ڕ
Name: ARABIC LETTER REH WITH SMALL V BELOW
Kurdish
UTF-8: 0xDA 0x95
UTF-16: 0x0695
C octal escaped UTF-8: \332\225
XML decimal entity: &#1685;
https://www.unicode.org/charts/PDF/U0600.pdf

![reh](https://user-images.githubusercontent.com/31791780/38118062-ed06c81a-33a7-11e8-952c-31a2cc10f579.png)

**Comment by behdad** [comment]
I agree this is not optimal, but I'm sure was done intentionally.  Makes sense for UI version. Not sure why done for non-UI.

@marekjez86 can you ask Kamal to review if this is acceptable form?

**Comment by pupyc** [comment]
I think even in the UI version it should not be there


(Noto Naskh Arabic UI)
![reh-noto-naskh-arabic-ui](https://user-images.githubusercontent.com/31791780/38163729-4c21c836-34e8-11e8-9e06-9bb88ced4aea.png)

---

(Noto Naskh Arabic)
![reh-noto-naskh-arabic](https://user-images.githubusercontent.com/31791780/38163731-4d4e26f0-34e8-11e8-910e-872f6c13ada2.png)

---
(Noto Naskh Arabic, Noto Naskh Arabic UI, Noto Sans Arabic, Noto Sans Arabic UI)

![all](https://user-images.githubusercontent.com/31791780/38163737-6d8ae3cc-34e8-11e8-86b5-d9119fcc9a39.png)
